study-builder/studies/basil/i18n/en.conf

```diff
@@ -256,242 +256,223 @@
+    "diffuse_astrocytoma": "Diffuse astrocytoma",
+    "diffuse_midline_glioma": "Diffuse midline glioma",
+    "embryonal_tumor_with_multilayered_rosettes": "Embryonal tumor with multilayered rosettes (ETMR)",
     "ependymoma": "Ependymoma",
-    "primitive_neuroectodermal_tumor_pnet": "Primitive neuroectodermal tumor (PNET)",
-    "cerebellar_astrocytoma_ca": "Cerebellar astrocytoma (CA)",
-    "cerebral_astrocytoma": "Cerebral astrocytoma",
-    "glioblastoma_gbm": "Glioblastoma (GBM)",
-    "glioma": "Glioma",
+    "ganglioglioma": "Ganglioglioma",
+    "germ_cell_tumor": "Germ cell tumor",
+    "glioblastoma_glioblastoma_multiforme": "Glioblastoma (glioblastoma multiforme) (GBM)",
+    "glioma_other": "Glioma, other",
+    "hemangioma": "Hemangioma",
     "medulloblastoma": "Medulloblastoma",
-    "neuroblastoma_nb": "Neuroblastoma (NB)",
+    "meningioma": "Meningioma",
+    "nerve_sheath_tumor": "Nerve sheath tumor",
+    "oligoastrocytoma": "Oligoastrocytoma",
     "oligodendroglioma": "Oligodendroglioma",
-    "pineal_astrocytoma": "Pineal astrocytoma",
-    "pituitary_adenoma": "Pituitary adenoma",
-    "visual_pathway_and_hypothalamic_glioma": "Visual pathway and hypothalamic glioma",
-    "brain_stem_cancer": "Brain stem cancer",
-    "breast_cancer": "Breast Cancer",
-    "triple_negative_breast_cancer": "Triple negative breast cancer",
-    "invasive_lobular_carcinoma": "Invasive lobular carcinoma",
-    "tubular_carcinoma": "Tubular carcinoma",
-    "invasive_cribriform_carcinoma": "Invasive cribriform carcinoma",
-    "medullary_carcinoma": "Medullary carcinoma",
-    "male_breast_cancer": "Male breast cancer",
-    "phyllodes_tumor": "Phyllodes tumor",
-    "inflammatory_breast_cancer": "Inflammatory Breast Cancer",
-    "metaplastic_carcinoma_of_the_breast": "Metaplastic carcinoma of the breast ",
-    "pagets_disease_of_the_breast": "Paget's Disease of the Breast",
-    "secretory_breast_carcinoma": "Secretory breast carcinoma ",
-    "salivary_gland_type_cancer_of_the_breast": "Salivary gland type cancer of the breast ",
-    "endocrine_cancer": "Endocrine cancer",
-    "adrenocortical_carcinoma": "Adrenocortical carcinoma",
-    "islet_cell_carcinoma_endocrine_pancreas": "Islet cell carcinoma (endocrine pancreas)",
-    "multiple_endocrine_neoplasia_syndrome": "Multiple endocrine neoplasia syndrome",
+    "optic_nerve_glioma": "Optic nerve glioma",
+    "pilocytic_astrocytoma": "Pilocytic astrocytoma (JPA)",
+    "pineal_tumor": "Pineal tumor",
+    "pituitary_tumor": "Pituitary tumor",
+    "pleomorphic_xanthoastrocytomas": "Pleomorphic xanthoastrocytomas (PXA)",
+    "primary_central_nervous_system_lymphoma": "Primary central nervous system lymphoma (PCNSL; CNS)",
+    "primitive_neuroectodermal_tumor": "Primitive neuroectodermal tumor (PNET)",
+    "schwannoma": "Schwannoma",
+    "group_prompt__breast_cancers": "Breast cancers",
+    "breast_cancer": "Breast Cancer (TNBC; MBC; IBC; ILC; IDLC; SBC; ICC)",
```

Comment:
Not seeing my previous comment. Let me try again:
An example of how I am expecting list option labels to look like:
Breast cancer
Breast adenocarcinoma
Triple negative breast cancer (TNBC)
Metastatic breast cancer (MBC)

Reply:
I don't think the spreadsheet is super clear about separating acronyms (abbreviations) from aliases, but not too difficult to figure out by reading the names what abbreviation goes with which alias.

Reply:
@MocanaAtBroad so it should be added to this breast cancer group all these options?

Reply:
Actually it's not quite clear to me how to interpret this content of the 2nd column for breast cancer:
```
breast adenocarcinoma; TNBC; triple negative breast cancer; metastatic breast cancer; MBC; breast carcinoma; Inflammatory Breast Cancer; IBC; Invasive Ductal Carcinoma; IDC; Invasive lobular carcinoma; ILC; mixed IDLC; IDLC; invasive mammary carcinoma; infiltrating mammary carcinoma; Metaplastic carcinoma of the breast; metaplastic; lobular; Paget's Disease of the Breast; mammary Paget's disease; Paget's disease of the nipple; Paget's disease of the nipple and areola; Secretory breast carcinoma; SBC; juvenille breast carcinoma; Tubular carcinoma; Medullary carcinoma; Invasive cribriform carcinoma; ICC; male breast cancer
```